internal/manifests/manifests_test.go

```diff
@@ -37,7 +37,7 @@ var _ = Describe("ClusterManifestTests", func() {
 		ctrl          *gomock.Controller
 		mockS3Client  *s3wrapper.MockAPI
 		dbName        = "cluster_manifest"
-		content       = "aGVsbG8gd29ybGQhCg=="
+		content       = "LS0tCm1lc3NhZ2U6IGhlbGxvIHdvcmxkCnVuaXQtdGVzdHM6IGFyZSBncmVhdAo="
```

Comment:
I think it would be nicer to have the real content and you just base64 encode it.